ios/native/Info.plist

```diff
@@ -36,7 +36,7 @@
 		</dict>
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string/>
+	<string></string>
```

Comment:
This change seems pointless

Reply:
This looks like it is something automatic that xcode did or formatting did. I didn't touch it but will try change it back!

Reply:
Solved!